mix.exs

```diff
@@ -71,4 +72,10 @@ defmodule Membrane.FFmpeg.VideoFilter.Mixfile do
       nest_modules_by_prefix: [Membrane.FFmpeg.VideoFilter]
     ]
   end
+
+  defp copy_images(_) do
+    File.cp_r("readme", "doc", fn source, destination ->
```

Comment:
Well, now it's broken on github: https://github.com/membraneframework/membrane_ffmpeg_video_filter_plugin/tree/develop

```suggestion
    File.cp_r("readme", "doc/readme", fn source, destination ->
```